geemap/common.py

```diff
@@ -6754,6 +6752,10 @@ def zonal_stats(
     if out_file_path is None:
         out_file_path = os.path.join(os.getcwd(), "zonal_stats.csv")
 
+    if "statistics_type" in kwargs:
+        stat_type = kwargs["statistics_type"]
```

Comment:
simplify. Combine the pop with the assignment

```
stat_type = kwargs.pop("statistics_type")
```

Reply:
Good suggestion. Fix in [here](https://github.com/gee-community/geemap/pull/1861/commits/c6f62ccc72cd47bf779713f3132d1f1dbccbf906).